credentials/credentials.go

```diff
@@ -64,7 +64,7 @@ type Credentials interface {
 	// be used for timeout and cancellation.
 	// TODO(zhaoq): Define the set of the qualified keys instead of leaving
 	// it as an arbitrary string.
-	GetRequestMetadata(ctx context.Context) (map[string]string, error)
+	GetRequestMetadata(ctx context.Context, audience ...string) (map[string]string, error)
```

Comment:
This seems weird, and deserving of more comments. The doc comment on this interface says this is common across all supported credentials, but as far as I know this "audience" concept is OAuth-specific.